src/main/resources/frontend/src/views/StudentView/SubmissionInfo.vue

```diff
@@ -32,6 +49,22 @@ const { submission } = defineProps<{
       <span v-else>passed <i class="fa-solid fa-circle-check" style="color: green"/></span>
     </p>
 
+    <div v-if="useAuthStore().user?.role == 'ADMIN' && commitVerificationFailed(submission)">
+      <InfoPanel id="approveSubmission">
+        <h4>Approve Blocked Submission</h4>
+        <p>This submission was blocked because it did not meet the git commit requirements.</p>
+        <p>Meet with the student and explain the importance of frequent and consistent commits.</p>
+        <p>You may, at your discretion, deduct 10% if it looks like the student is not learning the value/habit of repeated commits</p>
+        <div id="approvalButtons" v-if="unapproved">
+          <button @click="approve(true, $emit)">Approve with penalty</button>
+          <button @click="approve(false, $emit)" class="small" style="font-weight: normal; font-size: 0.9rem">Approve without penalty</button>
```

Comment:
Are these buttons at slightly different heights?

Reply:
Yes. Per my request, we've designed the system to de-emphasize approving without a penalty, although the option is provided.